internal/kldopenapi/abi2swagger.go

```diff
@@ -379,6 +391,7 @@ func (c *ABI2Swagger) addCommonParams(op *spec.Operation, isPOST bool, isConstru
 	privateForParam, _ := spec.NewRef("#/parameters/privateForParam")
 	privacyGroupIdParam, _ := spec.NewRef("#/parameters/privacyGroupIdParam")
 	registerParam, _ := spec.NewRef("#/parameters/registerParam")
+	optionParam, _ := spec.NewRef("#/parameters/optionParam")
```

Comment:
could be just me, I find `optionParam` vs `kld-opts` as query param kinda non-intuitive. I believe most other options try to maintain same name

Reply:
now that I think about that, single it's always a single value parameter, `option` sounds better than `opts`. Now it should be consistent after the latest commit.